js/pivx_shielding.js

```diff
@@ -29,19 +44,46 @@ export default class PIVXShielding {
     );
   }
 
-  constructor(shieldMan, extsk, isTestNet, blockHeight, commitmentTree) {
+  constructor(shieldMan, extsk, isTestNet, commitmentTree) {
+    /**
+     * Webassembly object that holds Shield related functions
+     * @private
+     */
     this.shieldMan = shieldMan;
+    /**
+     * Extended spending key
+     * @type {String}
+     * @private
+     */
     this.extsk = extsk;
+    /**
+     * Number of generated addresses
+     * @type {Number}
+     * @private
+     */
     this.generatedAddresses = 0;
+    /**
+     * @type {Boolean}
+     * @private
+     */
     this.isTestNet = isTestNet;
-    this.lastBlock = blockHeight;
```

Comment:
Why did you remove the lastBlock? It could be good for consistency check

Reply:
We aren't using it, if we need in the future we can just re-add this one line